version.go

```diff
@@ -3,7 +3,7 @@ package dns
 import "fmt"
 
 // Version is current version of this library.
-var Version = v{1, 1, 53}
```

Comment:
without saying yes or no to this pr, please revert this version bump from this pr

Reply:
Sure.